### PR TITLE
New Icon: Fairphone

### LINKED ISSUE #8262: fairphone icon
### Brand Name

fairphone

### Website

https://www.fairphone.com

### Popularity Metric

image

### Official Resources for Icon and Color

Fairphone_logo

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by ripzerotwo**
**Issue:** closes #8262 

**Similarweb rank:**54,264 (https://www.similarweb.com/website/fairphone.com/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Preview...
fairphone

### CODE REVIEW

**Comment by adriens** [comment]
Thanks a lot for the PR :pray:

**Comment by adriens** [comment]
Hi guyes, any chance to get this merged soon ?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3707,6 +3707,11 @@
             "hex": "EC1C24",
             "source": "https://facepunch.com/img/brand/default-light.svg"
         },
+        {
+            "title": "Fairphone",
+            "hex": "2FA6DF",
+            "source": "https://www.fairphone.com/"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:3713]:
```suggestion
            "source": "https://www.fairphone.com"
```
Remove trailing slash

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Merge latest develop into your PR and remove trailing slash

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [inline _data/simple-icons.json:3713]:
```suggestion
            "source": "https://www.fairphone.com",
            "license":  {
                "type": "CC-BY-NC-SA-4.0"
            }
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3707,6 +3707,11 @@
             "hex": "EC1C24",
             "source": "https://facepunch.com/img/brand/default-light.svg"
         },
+        {
+            "title": "Fairphone",
+            "hex": "2FA6DF",
```
Comment (adamrusted) [inline _data/simple-icons.json:3712]:
```suggestion
            "hex": "4495D1",
```
This hex is currently used in their logo.

File: icons/fairphone.svg
Comment (adamrusted) [inline icons/fairphone.svg:1]: I'm spotting a [few differences](https://github.com/simple-icons/simple-icons/assets/1830867/06c94ada-8426-4555-b602-a69014724c17) between your PR in black, and the source image in red. Would you mind giving this another go @ripzerotwo?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/fairphone.svg
Comment (adriens) [reply icons/fairphone.svg:1]: :pray: :crossed_fingers:

File: icons/fairphone.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...3h.2c-.1.804-.2 1.607-.2 2.41h.803c0-.301 0-.602.1-.903h.302c.703 0 1.205-.402 1.305-1.105 0-.703-.401-1.105-1.104-1.105m3.014 1.205h-.904c0-.401.1-.803.1-1.205h-.803c-.1 1.004-.302 2.11-.302 3.113h.804c0-.401.1-.803.1-1.205h.904c0 .402-.1.804-.1 1.205h.803c.1-1.004.2-2.109.301-3.113h-.803c0 .402-.1.804-.1 1.205m6.125.402a11.99 11.99 0 0 1-.803-1.607q-.402.1-.703.1c-.1 1.005-.302 2.01-.302 3.114h.804c0-.502.1-1.105.1-1.607l.904 1.506c0 .1.1.1.2.1h.603c.1-1.003.201-2.108.302-3.112h-.804A4.03 4.03 0 0 0 20.385 12M.904 11.699c0-.201 0-.402.1-.603H2.41c0-.2.1-.502.1-.703H.1c0 .201-.1.502-.1.703h.2C.1 12 0 12.803 0 13.606h.803c0-.401.1-.803.1-1.204h.804c.1 0 .201-.1.201-.201.1-.201.1-.301.1-.502H.904Zm21.59 1.205v-.502h.903c.1 0 .201-.1.201-.201v-.402h-1.104c0-.2 0-.301.1-.502H23.9c0-.2.1-.502.1-.703h-2.31c0 .201-.1.502-.1.703h.2c-.1.803-.2 1.607-.2 2.41h2.109c0-.2 0-.502.1-.703h-1.305v-.1Zm-5.122-.804c-.1.603-.3.804-.703.804s-.602-.301-.502-.904v-.1c.1-.503.402-.904.703-.904.201 0 .302.1.402.2l-.1.101c.1.301.2.502.2.803m.402-1.205.1-.1c-.2-.2-.502-.402-.903-.402-.804 0-1.507.703-1.607 1.607v.1c-.1.904.502 1.607 1.305 1.607s1.406-.602 1.507-1.506v-.1c.1-.503-.1-1.005-.402-1.206"/></svg>
```
Comment (PratyushJayachandran) [inline icons/fairphone.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Fairphone</title><path d="M3.25 12.358c.124-.306.25-.612.38-.918.058.306.12.613.185.918H3.25zm.129-1.943a46.426 46.426 0 0 0-1.396 3.151h.649a.234.234 0 0 0 .216-.154c.05-.137.102-.273.155-.41h.957l.103.411a.2.2 0 0 0 .195.155h.667a38.402 38.402 0 0 1-.752-3.152l-.794-.001zm1.935 3.152h.784c.062-1.044.17-2.089.321-3.134h-.783a36.666 36.666 0 0 0-.322 3.134m3.23-2.018c-.029.25-.201.407-.467.407h-.388c.028-.273.06-.546.094-.819h.384c.262 0 .41.134.378.402v.009m.793-.04c0-.008.002-.016.003-.023.037-.318-.026-.56-.172-.734-.17-.206-.461-.318-.871-.318H6.882c-.035.234-.067.468-.096.703h.218c-.103.81-.178 1.62-.226 2.43h.78c.02-.323.043-.646.07-.97h.278c.128.285.26.57.394.856a.2.2 0 0 0 .183.115h.756a53.18 53.18 0 0 1-.542-1.137 1.11 1.11 0 0 0 .64-.922m2.093.062c-.031.264-.212.43-.487.43h-.274c.03-.293.062-.572.099-.865h.27c.283 0 .427.134.393.42l-.001.014m-.257-1.137H9.866c-.033.23-.066.473-.096.703h.217a36.478 36.478 0 0 0-.224 2.432h.78c.018-.303.04-.606.065-.909h.3c.669-.009 1.225-.367 1.31-1.12l.002-.017c.083-.703-.326-1.088-1.046-1.088m3.032 1.19h-.866c.044-.396.096-.793.154-1.19h-.784a36.105 36.105 0 0 0-.32 3.134h.784c.023-.403.053-.806.09-1.209h.867c-.037.403-.067.806-.09 1.21h.788a36.75 36.75 0 0 1 .321-3.135h-.789c-.058.397-.11.794-.154 1.19m6.068.386a52.412 52.412 0 0 1-.847-1.631l-.718.12a36.018 36.018 0 0 0-.312 3.07h.773c.031-.547.076-1.095.133-1.642.278.516.566 1.032.865 1.548a.188.188 0 0 0 .163.094h.59a36.75 36.75 0 0 1 .32-3.134h-.772c-.076.526-.142 1.051-.196 1.576M.941 11.713c.02-.185.042-.37.066-.556h1.427c.031-.24.064-.482.1-.724H.104c-.036.242-.069.483-.1.725h.218A35.93 35.93 0 0 0 0 13.568h.784c.023-.386.052-.773.087-1.16h.845a.247.247 0 0 0 .236-.221c.016-.158.032-.315.05-.473H.94zm21.461 1.147c.015-.18.03-.359.047-.538h.859a.247.247 0 0 0 .237-.221c.014-.149.03-.297.047-.446h-1.075c.02-.172.04-.343.062-.515h1.324c.03-.235.063-.471.097-.707h-2.317c-.035.236-.067.471-.097.707h.218a35.363 35.363 0 0 0-.225 2.427h2.119c.013-.236.03-.471.048-.707h-1.344zm-5.079-.795-.005.049c-.054.542-.293.782-.66.782-.36 0-.59-.332-.535-.874 0-.016.003-.033.005-.049.056-.542.349-.869.71-.869a.46.46 0 0 1 .408.225l-.11.106c.093.15.217.357.188.63m.414-1.202.108-.105c-.212-.243-.52-.383-.903-.383-.818 0-1.52.658-1.616 1.593l-.007.068c-.095.94.463 1.59 1.286 1.59.822 0 1.419-.572 1.513-1.508l.007-.067a1.545 1.545 0 0 0-.388-1.188"/></svg>
```
This should fix the issues

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:3713]: @adamrusted  how come the license for the logo is CC-BY-NC-SA-4.0?

Comment (adamrusted) [reply _data/simple-icons.json:3713]: We can remove the license, as I can't recall where I saw it.

**Review by adamrusted — APPROVED** [review]

**Comment by adriens** [comment]
Hi, when will the icon be available please ?

https://simpleicons.org/?q=fairphone

image

**Comment by adriens** [comment]
(I'm quite impatient to add it to target project ;-p)

**Comment by PratyushJayachandran** [comment]
> (I'm quite impatient to add it to target project ;-p)

After weekly release on upcoming Sunday.

**Comment by adriens** [comment]
Great, thanks a lot for the fast and kind answers

**Pull request merged**